ui/src/actions/login.js

```diff
@@ -95,10 +95,9 @@ export function logIn(proposal, password) {
 
 export function signOut() {
   return async (dispatch) => {
-    await sendSignOut();
-    dispatch({ type: 'SIGNOUT' });
```

Comment:
I don't think this action is used in any of the reducers. One could argue that the `login` reducer should listen to this action to reset the `loginInfo` state... but this is not [a pattern](https://redux.js.org/style-guide/#model-actions-as-events-not-setters) that is used in MXCuBE so far, so I prefer to remove the `SIGNOUT` action for now.